public/externalLibs/sound/sounds.js

```diff
@@ -60,7 +60,7 @@ function discretize_from(wave, duration, elapsed_duration, sample_length, data)
 // Quantize real amplitude values into standard 4-bit PCM levels
 function quantize(data) {
     for (var i = 0; i < data.length; i++) {
-        data[i] = Math.round((data[i] + 1) * 126);
+        data[i] = Math.round(data[i] * 32767);
```

Comment:
why not "+ 1"? what is the range for the result?

Reply:
The range is [-32768, 32767] (see riffwave.js)

(added a comment as well)